Content.Server/GameObjects/Components/Items/Storage/EntityStorageComponent.cs

```diff
@@ -424,6 +422,7 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
             }
 
             _beingWelded = false;
+            CanWeldShut = true;
```

Comment:
i thought this probably wasnt an ideal solution so looked through the code and here are my thoughts:

the issue is that the StroageVisualLayers.Welded layer requires that StorageVisuals.CanWeld is true, which is only set to true when CanWeld is set to true (which it never was), so this change does indeed fix that but there could be a more optimal solution (not saying you necessarily have to do that in this PR)

CanWeldShut shouldn't be getting 'set' ever really, IMO it should just be loaded initially when the entity spawns, like lockers can be welded shut but crates can't, so having behavior depend on it being set is a little weird. The reason it even changes this CanWeld appearance data is because the StorageVisualizer uses it to determine if it should change the layer. A good solution would be to move the 'if appearance component set storagevisuals.canweld to value' bit to the Initialize instead of the setter for CanWeld and remove the setter for CanWeld entirely

Reply:
I ended up moving the if statement to Initialize as EntityStorageComponent did not have a startup and that seems to have worked. Not sure if that creates any issues but it seems similar enough to Startup.